packages/@azure/eventhubs/client/package.json

```diff
@@ -19,13 +19,13 @@
     "node": ">=6.0.0"
   },
   "dependencies": {
-    "@azure/amqp-common": "^1.0.0-preview.1",
+    "@azure/amqp-common": "git+https://github.com/Azure/amqp-common-js.git#master",
```

Comment:
The new version of amqp common is now released.
Can you update this to `"^1.0.0-preview.2`?